lib/Calendar/calendar.dart

```diff
@@ -63,18 +67,20 @@ class CalendarState extends State<Calendar> {
     _location = "";
     shouldShowForm = false;
     super.initState();
-    _fetchDataAsync();
```

Comment:
na een delete halen we niet de data opnieuw op? werkt dat nog wel?

Reply:
Dit hebben we net gefixed samen